.travis.yml

```diff
@@ -0,0 +1,14 @@
+branches:
+  except:
+    - master
+language: ruby
+cache: bundler
+sudo: false
+rvm:
```

Comment:
I do have special characters in the key though, "7.2" for example. "7.2": "value" translates to a symbol key (and there is no way to specify a string key without a rocket, but I need to rewrite a lot of stuff to expect strings instead of symbols now anyways)